pyCGM_Single/pyCGM.py

```diff
@@ -139,24 +136,26 @@ def pelvisJointCenter(frame):
 
     return pelvis
 
-def hipJointCenter(frame,pel_origin,pel_x,pel_y,pel_z,vsk=None):
-    """Calculate the hip joint center function.
 
-    Takes in a dictionary of x,y,z positions and marker names, as well as an index.
-    Calculates the hip joint center and returns the hip joint center.
+def hipJointCenter(frame, pel_origin, pel_x, pel_y, pel_z, vsk=None):
+    u"""Calculate the hip joint center.
+
+    Takes in a dictionary of x,y,z positions and marker names, as well as an
+    index. Calculates the hip joint center and returns the hip joint center.
 
     Other landmarks used: origin, sacrum
-    Subject Measurement values used: MeanLegLength, R_AsisToTrocanterMeasure, InterAsisDistance, L_AsisToTrocanterMeasure
 
-    Hip Joint Center: Computed using Hip Joint Center Calculation (ref. Davis_1991)
+    Subject Measurement values used: MeanLegLength, R_AsisToTrocanterMeasure,
+    InterAsisDistance, L_AsisToTrocanterMeasure
+
+    Hip Joint Center: Computed using Hip Joint Center Calculation [1]_.
 
     Parameters
     ----------
-    frame : dict
+    frame : dict, optional
```

Comment:
frame is not optional, it is a required positional argument.  It just happens to not be used in this case and should not be an argument at all, but we need to fix that in the future.